### PR TITLE
fix: Missing some translation in hindi language #987

### LINKED ISSUE #987: Missing some translation in hindi language
**Summary:**

The application is missing some translation in Hindi language

**Steps to reproduce the issue:**

1. Go to settings.
1. Change language to hindi.

**Expected behaviour:**

All strings should be translated to Hindi.  

**Observed behaviour:**

Some strings do not have Hindi tranlation.

**Device and Android version:**

Vivo 1724 
Android 8.1.0
Funtouch OS 4.0

### PR DESCRIPTION
**Pull request by ShivangiSingh17**
Fixes #987 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by ShivangiSingh17** [comment]
@miPlodder,@therajanmaurya can you please review this PR.

File: app/src/main/res/values-hi/strings.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
     <string name="account_balance">खाते में शेष</string>
     <string name="total_deposits">कुल जमा</string>
     <string name="open_drawer">Open Drawer</string>
-    <string name="close_drawer">Close Drawer</string>
+    <string name="close_drawer">दराज को बंद करें</string>
```
Comment (therajanmaurya) [inline app/src/main/res/values-hi/strings.xml]: This should not be translated, translate=false

File: app/src/main/res/values-hi/strings.xml [hunk 2]
```diff
@@ -168,15 +168,15 @@
...
+    <string name="view_qr_code">इस खाते के लिए क्यू.आर कोड देखें</string>
     <string name="last_trans">पिछले लेनदेन का विवरण</string>
     <string name="passcode_setup">लॉगिन करने के लिए एक पासकोड सेटअप करें</string>
     <string name="error_fetching_third_party_transfer_template">थर्ड पार्टी ट्रांसफर टेम्प्लेट को लाने में त्रुटि</string>
     <string name="choose">एप्लिकेशन चुनें</string>
 
     <string name="fetching_client">क्लाइंट ला रहा है</string>
 
-    <string name="not_contain_username">Spaces</string>
+    <string name="not_contain_username">रिक्त स्थान</string>
```
Comment (therajanmaurya) [inline app/src/main/res/values-hi/strings.xml]: Please check in which context this string is used in the application, I feel like It is just to follow non-hardcoded string pattern.

**Review by therajanmaurya — CHANGES_REQUESTED** [review]
Please before translating string, Please check string should be translated or it just to follow hard coded string pattern.

File: app/src/main/res/values-hi/strings.xml (hunk 1 above)
Comment (ShivangiSingh17) [reply app/src/main/res/values-hi/strings.xml]: Sorry I did that by mistake. I corrected it now.

File: app/src/main/res/values-hi/strings.xml (hunk 2 above)
Comment (ShivangiSingh17) [reply app/src/main/res/values-hi/strings.xml]: I resolved it now.

**Comment by ShivangiSingh17** [comment]
@therajanmaurya , I made the suggested changes. Can you please review the PR .

**Comment by ShivangiSingh17** [comment]
@therajanmaurya can you please review this PR, I made the suggested changes.

File: app/src/main/res/values-hi/strings.xml [hunk 3]
```diff
@@ -390,6 +387,74 @@
...
+    <string name="dialog_message_storage_permission_denied_prompt">भंडारण अनुमति के बिना आप लाभार्थी को जोड़ने के लिए क्यूआर कोड अपलोड नहीं कर पाएंगे। क्या आप सुनिश्चित हैं कि आप इस अनुमति को अस्वीकार करना चाहते हैं?</string>
+    <string name="dialog_message_read_storage_permission_never_ask_again">आपने भंडारण पढ़ने की अनुमति से इनकार कर दिया है, इस अनुमति के बिना आप क्यूआर कोड का उपयोग करके लाभार्थियों को नहीं जोड़ पाएंगे। कृपया इसे सेटिंग में सक्षम करें</string>
+    <string name="dialog_message_write_storage_permission_never_ask_again">आपने भंडारण करने के लिए लिखने की अनुमति से इनकार कर दिया है, इस अनुमति के बिना आप क्यूआर कोड का उपयोग करके लाभार्थियों को नहीं जोड़ पाएंगे। कृपया इसे सेटिंग में सक्षम करें</string>
+    <string name="permission_denied_storage">"भंडारण की अनुमति से इनकार "</string>
+    <string name="dialog_are_you_sure_that_you_want_to_string">क्या आप सुनिश्चित हैं कि आप% 1 $ s करना चाहते हैं?</string>
+    <string name="double_amount">%1$.2f</string>
+    <string name="clear_filters">फिल्टर साफ करे</string>
+    <string name="notification">सूचनाएं</string>
+    <string name="base_url">आधार यू.आर.ए्ल</string>
```
Comment (prashantkh19) [inline app/src/main/res/values-hi/strings.xml]: This string should not be translated.

File: app/src/main/res/values-hi/strings.xml [hunk 4]
```diff
@@ -390,6 +387,74 @@
...
+    <string name="tenant">किरायेदार</string>
+    <string name="message">संदेश</string>
+    <string name="no_notification">कोई अधिसूचना नहीं</string>
+    <string name="pref_base_url_title">अद्यतन समापन बिंदु</string>
+    <string name="pref_base_url_desc">अपना समापन बिंदु कॉन्फ़िगरेशन बदलने के लिए यहां क्लिक करें</string>
+    <string name="enter_base_url">आधार यू.आर.ए्ल दर्ज करें</string>
+    <string name="enter_tenant">किरायेदार दर्ज करें</string>
+    <string name="pref_configuration">वरीयता विन्यास</string>
+    <string name="contact_email">support@mifos.org</string>
```
Comment (prashantkh19) [inline app/src/main/res/values-hi/strings.xml]: This should be a non-translatable string

**Comment by prashantkh19** [comment]
@ShivangiSingh17 please update the PR with the above requested changes.

**Comment by ShivangiSingh17** [comment]
@prashantkh19 made the suggested changes.

**Comment by prashantkh19** [comment]
@ShivangiSingh17 can you rebase the branch and fix the travis build error?

**Comment by prashantkh19** [comment]
Squash your commits also.

**Comment by geekanamika** [comment]
@ShivangiSingh17 Closing this PR as there's no update in the recent few months over the review provided by @prashantkh19 :)

**Pull request closed without merging**